### PR TITLE
Feature/weee sent on remove site

### PR DESCRIPTION
**Pull request by SamGoodeySFW**

### CODE REVIEW

**Comment by SamGoodeySFW** [comment]
Just creating the pull request to get some initial thoughts, need to discuss with Lynne on Tuesday about potentially changing how we remove/delete things

File: src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnAndAatfToSentOnSummaryListViewModelMap.cs [hunk 1]
```diff
@@ -38,75 +38,78 @@ public SentOnSiteSummaryListViewModel Map(ReturnAndAatfToSentOnSummaryListViewMo
 
             foreach (var item in source.WeeeSentOnDataItems)
             {
-                var siteData = new WeeeSentOnSummaryListData()
+                if (item.Removed != true)
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnAndAatfToSentOnSummaryListViewModelMap.cs]: Remove the removed property as hard deleting now

**Comment by JohnPadgettCivica** [comment]
Remove the RemoveMany and Remove methods

File: src/EA.Weee.Database/scripts/Update/Release 2/Sprint8/20190502-1045-UpdateWeeeSentOnTableWithDeleteColumn.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿ALTER TABLE [AATF].[WeeeSentOn] ADD Removed BIT
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Database/scripts/Update/Release 2/Sprint8/20190502-1045-UpdateWeeeSentOnTableWithDeleteColumn.sql]: Delete this script and remove the column from your DB

**Pull request merged**